### PR TITLE
Implement toolkit Python Auth Samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by frankqianms — APPROVED** [review]
LGTM

File: samples/bot-teams-authentication/python/appManifest/manifest.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.19/MicrosoftTeams.schema.json",
   "manifestVersion": "1.19",
```
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/python/appManifest/manifest.json:3]: icons not needed for appManifest?

File: samples/msgext-search-auth-config/python/appManifest/manifest.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
   "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.19/MicrosoftTeams.schema.json",
   "manifestVersion": "1.19",
   "version": "1.0.0",
-  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "id": "${{TEAMS_APP_ID}}",
```
Comment (Pawank-MSFT) [inline samples/msgext-search-auth-config/python/appManifest/manifest.json:5]: icons not needed for appManifest?

**Review by Pawank-MSFT — COMMENTED** [review]
Please fix small comments

File: samples/bot-teams-authentication/python/appManifest/manifest.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-teams-authentication/python/appManifest/manifest.json:3]: Need FYI
image

File: samples/msgext-search-auth-config/python/appManifest/manifest.json (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/msgext-search-auth-config/python/appManifest/manifest.json:5]: Need FYI
image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, approving!

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/12179035402

**Pull request merged**